packages/@o3r/core/schematics/rule-factories/generic-updates/index.ts

```diff
@@ -1,9 +1,12 @@
 import type { Rule, SchematicContext, Tree } from '@angular-devkit/schematics';
+import { getPackageManager, WorkspaceSchema } from '@o3r/schematics';
 
 /**
  * Generic basic updates to be done on a repository
+ *
+ * @param workspaceConfig
  */
-export function genericUpdates(): Rule {
+export function genericUpdates(workspaceConfig?: WorkspaceSchema | null): Rule {
```

Comment:
This change should not be in `@o3r/workspace` package?
This package manager related gitignore seem to be a workspace rule only :S

Reply:
Indeed :)
but in this version we don;t have the 'o3r/workspace'. It was included in '>= 9.4'.
I'll take care of it in the cascading.

Reply:
ref: https://github.com/AmadeusITGroup/otter/pull/951